generated/nidaqmx/constants.py

```diff
@@ -627,6 +627,20 @@ class ShuntCalSelect(Enum):
     AAND_B = 12515  #: Switches A and B.
```

Comment:
Since we're bumping to 1.0 as part of this effort, I think fixes like this are OK. For methods we can use `deprecation.deprecated`, but I don't see anything similar for enums. At the very least we should mention the change in the CHANGELOG.md